package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-router-prompt",
-  "version": "0.6.1",
+  "version": "0.7.1",
+  "name": "@aptuitiv/react-router-prompt",
```

Comment:
Please keep the name and URL same in package json

Reply:
I didn't expect you to reply so quickly so I deployed this under my own package to use in our code. Thank you for reviewing so quickly.